dotnet.cmd

```diff
@@ -21,7 +21,7 @@ if NOT [%ERRORLEVEL%]==[0] (
 
```

Comment:
Kept `call %~dp0init-tools.cmd` above because we are still depending on some build tasks from Tools.  Most notable is `BinClashLogger`.